### ISSUE TITLE
3D U-Net: Learning Dense Volumetric Segmentation from Sparse Annotation

### ISSUE THREAD

**Issue by HYUNMIN-HWANG** [yskim0/One-Day-One-Paper#35]
# 3D U-Net: Learning Dense Volumetric Segmentation from Sparse Annotation
- https://arxiv.org/abs/1606.06650
- [Submitted on 21 Jun 2016]
- MICCAI 2016

---

## Summary
U-NET의 3D 버전 논문 !! 바이오 데이터 중 3D 데이터는 많지만 이를 직접 segmentation 하기에는 어려움이 있기 때문에 해당 논문에서 3D annotation 방법을 제안했다.

**해당 논문의 아이디어는 크게 두 가지 부분으로 나눌 수 있다.**
![image](https://user-images.githubusercontent.com/70581043/128590785-e84d252a-1b0c-4def-ac04-6a6c421e3364.png)
1. semi-automated setup, the user annotates some slices in the volume to be segmented. (2D 이미지 몇 개만 annotate 한 후 segmentation을 진행한다.)
2. fully-automated setup, we assume that a representative, sparsely annotated training set exists. (1에서 훈련한 모델을 기반으로 새로운 3D 이미지를 segmentation 한다.)

**모델 및 훈련 특징**
![image](https://user-images.githubusercontent.com/70581043/128590768-a2bce6b0-5dcc-4ebb-9da4-d33e9696e50f.png)
-  The network is based on the previous u-net architecture
- 단, 3D convolutions, 3D max pooling, and 3D up-convolutional layers 사용함
- use batch normalization
- weighted softmax loss function (unlabeld pixels는 weight를 0으로 세팅함, labeld data만 1)
- augmentation : rotation, scaling and gray value augmentation, smooth dense deformation

**결과**
1. Semi-Automated Segmentation
![image](https://user-images.githubusercontent.com/70581043/128590634-983f7b2f-b084-4eb9-a8f6-fd2844ef318d.png)
2d 이미지와 3d 비교했을 때, 3d(with BN)일 때 IOU가 가장 높았다. semi-automated segmentation 몇 장 했는지에 따른 결과를 비교했을 때 S3 성능이 가장 좋았다.

2. Fully-automated Segmentation
![image](https://user-images.githubusercontent.com/70581043/128590642-66359baf-1da7-4845-bece-7785ef7d6831.png)
BN를 할 때(test3을 제외하고) & 샘플 사이즈가 클 때 성능이 좋았다.
